torch/csrc/jit/mobile/interpreter.cpp

```diff
@@ -386,6 +386,9 @@ bool InterpreterState::run(Stack& stack) {
 }
 
 IValue& InterpreterState::reg(size_t reg) {
+  if (registers_.empty() || reg == 0 || reg > registers_.size()) {
+    TORCH_CHECK(false, "Invalid register index: ", reg);
+  }
```

Comment:
Nit (but also, please confirm the perf implication of this change, as this function can be on the hot path)
```suggestion
    TORCH_CHECK(!registers_.empty(), "Registers are empty");
    TORCH_CHECK(reg > 0 && reg =< registers_.size(), "Invalid register index: ", reg);
```